sdk/ai/azure-ai-projects/CHANGELOG.md

```diff
@@ -5,9 +5,12 @@
 ### Features added
 
 * New parameters added for Azure AI Search tool, with corresponding sample update.
+* Fabric tool REST name updated, along with convenience code.
```

Comment:
Let's check if Linda has any comments here.

We may want to mention the Fabric tool is under preview and requires private preview access or smth like that.